Cargo.toml

```diff
@@ -24,12 +24,12 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [dependencies]
-algebra = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "development", features = ["parallel", "fft", "derive"] }
-bench-utils = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "development" }
-r1cs-core = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "development" }
-r1cs-std = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "development" }
+algebra = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "refactor_constraint_system", features = ["parallel", "fft", "derive"] }
+bench-utils = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "refactor_constraint_system" }
+r1cs-core = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "refactor_constraint_system" }
+r1cs-std = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "refactor_constraint_system" }
```

Comment:
@lgiussan  Is  the refactoring of the constraint system in ginger-lib not yet merged into development?

Reply:
Not yet. Approved by Daniele but still not merged.